app/services/articles/feeds/large_forem_experimental.rb

```diff
@@ -7,6 +7,15 @@ def initialize(user: nil, number_of_articles: Article::DEFAULT_FEED_PAGINATION_W
         @page = page
         @tag = tag
         @article_score_applicator = Articles::Feeds::ArticleScoreCalculatorForUser.new(user: user)
+
+        # Per #15292 we want to move towards not requiring feature's
```

Comment:
I sympathize with this typo

```suggestion
        # Per #15292 we want to move towards not requiring features
```